src/main/java/com/akademia/projectplanner/validation/RegistrationValidator.java

```diff
@@ -4,7 +4,7 @@
 
 public final class RegistrationValidator {
 
-  private RegistrationValidator() {
+  RegistrationValidator() {
```

Comment:
wcześniej nie zwróciłem uwagi na to więc prośba o refaktor:
- usuń proszę ten throw exception i zostaw pusty konstruktor jako private (ze względu na statyczność metod)
- zrób testy do klasy klasę RegistrationServiceImpl gdzie rzucamy wyjątki: tak jak zrobiłaś test jest ok (sprawdzamy treść komunikatu) ale zrób jeszcze drugi test na samo rzucanie poprawnego exceptiona - wtedy z użyciem assertThrow :)